movies/src/test/java/comp.fundacionjala/movies/CustomerTest.java

```diff
@@ -0,0 +1,85 @@
+package comp.fundacionjala.movies;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static comp.fundacionjala.movies.Constants.DELTA;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class CustomerTest {
+
+    private static final String customerName = "Juan";
+
+    private static Rental rentalChildrenMovie;
+
+    private static Rental rentalRegularMovie;
+
+    private static Rental rentalNewReleaseMovie;
+
+    private static int daysRented = 2;
+
+    private static Customer customer;
+
+    @BeforeClass
+    public static void setUp() {
+        String childrenMovieTitle = "Resident Evil I";
+        String regularMovieTitle = "Resident Evil II";
+        String newReleaseMovieTitle = "Resident Evil III";
+        rentalChildrenMovie = new Rental(new ChildrenMovie(childrenMovieTitle), daysRented);
```

Comment:
@carledriss 
the method setup() requires  that this variables(childernMovie,regularMovie, and newReleaseMovie) uses the static modificator